test/test_xmath.cpp

```diff
@@ -692,4 +692,27 @@ namespace xt
         EXPECT_EQ(count_nonzeros(a, {0}, evaluation_strategy::immediate()), ea0);
         EXPECT_EQ(count_nonzeros(a, {1}, evaluation_strategy::immediate()), ea1);
     }
+
+    TEST(xmath, trapz)
+    {
+        xt::xarray<int> a = {{0, 1, 2},
+                             {3, 4, 5}};
+        auto res1 = trapz(a, 1.0, 0);
+        EXPECT_EQ(res1.size(), 3);
```

Comment:
Just a small remark: you can also compare against other xarrays like this:

```
xt::xarray<double> expected = {1.5, 2.5, 3.5};
EXPECT_EQ(res1, expected);
```

which should compare the shape and the values.

Reply:
Cool, thks. I'll modify it